code/src/components/Survey.js

```diff
@@ -0,0 +1,149 @@
+import React, {useState} from 'react';
+
+import {TextInput} from './TextInput';
+import {RadioInput} from './RadioInput';
+import SelectInput from './SelectInput';
+import ProgressBar from './ProgressBar';
+
+
+
+export const Survey = () => {
+
+    //STATES 
+
+    const [name, setName] = useState('');
+    const [city, setCity] = useState('');
+    const [housing, setHousing] = useState('');
+    const [footprint, setFootprint] = useState('');
+    const [lights, setLights] = useState('');
+
+    const [isSubmit, setIsSubmit] = useState(false);
+    const [section, setSection] = useState('question1');
+
+
+    // FUNCTIONS 
+
+    //Shows the result page when submit button is clicked
+    const handleSubmit = (event) => {
+        event.preventDefault()
+        setIsSubmit(true);
+        window.location.href="#results";
+    }
+
+    //Show the form again when button is clicked on result page
+    const showForm = () => {
+        setIsSubmit(false);
+        setSection('question1')
+    }
+
+    //Show the next question if the questions has been answered 
+    const showNextQuestion = (next,value) => {
+        if (value !== '') {
+        setSection(next)
+        }
+    } 
+
+    //THE RENDERS
+
+    // Shows form if it is not submitted. 
+    if (isSubmit === false){
+        return (
+            <>
+            
+                <form className="main-form" onSubmit={handleSubmit} id="main-form">
+                    <ProgressBar currentSection={section}/>
+
+                    {/* Renders questions with conditional rendering depending on the section value */}
+                    {section === 'question1' && (
+                    <>
```

Comment:
Do you need to add the React Fragments on line 58 and following since you're wrapping the whole form already with the tags between line 51 and 128?